src/spot.gresource.xml

```diff
@@ -13,11 +13,11 @@
     <!-- artist -->
     <file alias="components/artist.ui">app/components/artist/artist.ui</file>
     <!-- details -->
-    <file alias="components/details.css">app/components/details/details.css</file>
     <file alias="components/details.ui">app/components/details/details.ui</file>
+		<file alias="components/album_header.ui">app/components/details/album_header.ui</file>
+		<file alias="components/album_header.css">app/components/details/album_header.css</file>
```

Comment:
file does not exist apparently, forgot to commit it? removed it in the meantime maybe?

Reply:
Apparently :\
This also fixes the issues you commented on the PR itself.